core/kernel/tee_ta_manager.c

```diff
@@ -277,6 +278,43 @@ static TEE_Result check_client(struct tee_ta_session *s, const TEE_Identity *id)
 	return TEE_SUCCESS;
 }
 
+/* check if invocation parameters matches TA properties */
+static TEE_Result check_params(struct tee_ta_session *s,
```

Comment:
Why using `TEE_Result` instead of `bool` as we any way intentionally replace anything that isn't `TEE_SUCCESS`?